app/models/qdm/encounter_performed.rb

```diff
@@ -17,5 +17,16 @@ class EncounterPerformed < DataElement
     field :qdmCategory, type: String, default: 'encounter'
     field :qdmStatus, type: String, default: 'performed'
     field :qdmVersion, type: String, default: '5.4'
+
+    def shift_years(year_shift)
```

Comment:
This is a generated file. If we want this function to be added to this class, it will need to be added in the generator.

Reply:
Hmm, when i run `./bin/validate_generator.sh` locally it exits with 1, but somehow travis didn't fail?